strapi/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:18-alpine
+# Installing libvips-dev for sharp Compatibility
+RUN apk update && apk add --no-cache build-base gcc autoconf automake zlib-dev libpng-dev nasm bash vips-dev
+ARG NODE_ENV=development
+ENV NODE_ENV=${NODE_ENV}
+
+WORKDIR /opt/
+COPY package.json package-lock.json ./
+RUN npm config set fetch-retry-maxtimeout 600000 -g && npm install
```

Comment:
There's `npm ci` which is supposedly designed specifically for CI. Though last time I used it (a long time ago) the install was failing. But might be worth a try.